cob_cartesian_controller/include/cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_sinoid.h

```diff
@@ -0,0 +1,54 @@
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_cartesian_controller
+ *
+ * \author
+ *   Author: Christoph Mark, email: christoph.mark@ipa.fraunhofer.de / christoph.mark@gmail.com
+ *
+ * \date Date of creation: December, 2015
+ *
+ * \brief
+ *
+ ****************************************************************/
+#ifndef COB_CONTROL_COB_CARTESIAN_CONTROLLER_INCLUDE_COB_CARTESIAN_CONTROLLER_TRAJECTORY_PROFILE_GENERATOR_TRAJECTORY_PROFILE_GENERATOR_SINOID_H_
+#define COB_CONTROL_COB_CARTESIAN_CONTROLLER_INCLUDE_COB_CARTESIAN_CONTROLLER_TRAJECTORY_PROFILE_GENERATOR_TRAJECTORY_PROFILE_GENERATOR_SINOID_H_
+
+#include "cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_base.h"
+
+/* BEGIN TrajectoryProfileSinoid ****************************************************************************************/
+class TrajectoryProfileSinoid : public TrajectoryProfileBase
+{
+    public:
+        TrajectoryProfileSinoid(const cob_cartesian_controller::CartesianActionStruct& params)
+        : TrajectoryProfileBase(params)
+        {        }
+
+        ~TrajectoryProfileSinoid() {}
+        virtual cob_cartesian_controller::ProfileTimings getProfileTimings(double Se, double te, double accl, double vel, bool calcMaxTe);
```

Comment:
this would need to be adjusted according to comment https://github.com/ipa320/cob_control/pull/74/files#r47380086